plugins/wasm-go/Dockerfile

```
@@ -13,9 +13,8 @@ COPY . .
 
 WORKDIR /workspace/extensions/$PLUGIN_NAME
 
-RUN go mod tidy
-RUN \
-    GOOS=wasip1 GOARCH=wasm go build -buildmode=c-shared -o /main.wasm ./ 
+RUN go mod tidy -v
```

```suggestion
RUN go mod tidy
```
`go mod tidy` 应该不太需要显示详细信息，可以保留原样，自己本地需要 debug 的时候可以加上 `-v` 参数。

---

OK， 我调整回去